webCommander/centralHub/views.py

```diff
@@ -51,3 +61,15 @@ def get_artists(request):
         "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
     )
     return HttpResponse(options)
+
+def get_artist_choices(request):
+    print(request.GET.get('artist', ""))
```

Comment:
seems you're doing a POST here:

<img width="1582" alt="image" src="https://github.com/abaddonpuff/commandCenter/assets/387927/67941272-2c2e-46f6-b818-b0fdca5fbc3d">

Reply:
as per form:

```
    <form method="post" action="{% url 'get_artist_choices' %}" hx-post="{% url 'get_artist_choices' %}" hx-target="#artistChoicesContainer">
```

Reply:
I put a breakpoint here to debug that:

<img width="577" alt="image" src="https://github.com/abaddonpuff/commandCenter/assets/387927/f4cdf3f3-9144-4578-ac09-0e33cbd642f8">

Reply:
from there indeed something wrong with the return rendering:

<img width="923" alt="image" src="https://github.com/abaddonpuff/commandCenter/assets/387927/60b56942-10e9-4d22-882f-e30012a8e136">

Reply:
which is caused by the render below ...